File: leetcode/day3/python/main.py

import so
from typing import List


def main():
    target = 5
    nums = [1, 2, 3, 4, 5, 6, 7, 8, 9]

    s = so.Solution()
    res = s.twoSum(nums, target)
    print("result", res)


if __name__ == "__main__":
    main()

File: leetcode/day3/cpp/main.cpp

#include "so.h"
#include <iostream>
#include <string>
#include <vector>

int main(int argc, char **argv)
{
	Solution s{};
	std::vector<int> nums = {1, 2, 3, 4, 5, 6, 7, 8, 9};
	std::vector<int> res = s.twoSum(nums, 5);

	for (size_t i = 0; i < res.size(); i++)
	{
		/* code */
		std::cout << res[i] << std::endl;
	}

	return 0;
}
